src/server/service/kv.rs

```diff
@@ -1153,15 +1153,20 @@ fn future_get<E: Engine, L: LockManager>(
     storage: &Storage<E, L>,
     mut req: GetRequest,
 ) -> impl Future<Output = ServerResult<GetResponse>> {
+    let mut statistics = Statistics::default();
     let v = storage.get(
         req.take_context(),
         Key::from_raw(req.get_key()),
         req.get_version().into(),
+        &mut statistics,
     );
 
     async move {
+        let perf_statistics = PerfStatisticsInstant::new();
```

Comment:
`PerfStatisticsInstant` is a thread local variable containing monotonically increasing counters. I think you need to use two `PerfStatisticsInstant` and return their delta via `PerfStatisticsDelta` instead.